wgpu-native/src/command/transfer.rs

```diff
@@ -86,7 +86,10 @@ pub fn command_encoder_copy_buffer_to_buffer<B: GfxBackend>(
         cmb.trackers
             .buffers
             .use_replace(&*buffer_guard, source, (), BufferUsage::COPY_SRC);
-    assert!(src_buffer.usage.contains(BufferUsage::COPY_SRC));
+    assert!(
+        src_buffer.usage.contains(BufferUsage::COPY_SRC),
+        "Source buffer does not have the `COPY_SRC` usage flag"
```

Comment:
Do you think these messages are useful? It reads to me like a rewording of the same line `src_buffer.usage.contains(BufferUsage::COPY_SRC)`

Reply:
As a Rust developer familiar with the `bitfields` crate, the line `src_buffer.usage.contains` is pretty understandable, but from the perspective of a C developer it wouldn't be (assuming they use wgpu-rs and don't understand its internals). IMO a language-agnostic message is more understandable.

Reply:
It literally says "buffer usage contains COPY_SRC", I think it's clear even for non-rustaceans.
More importantly, the asserts are a stopgap, we'll need to change them into proper error handling, likely very soon. So we shouldn't invest in making the messages super clear, or at least not within this PR.

Reply:
I've removed them from this PR